src/components/LabelledLoader.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import Spinner from 'svelte-spinner';
+    export let languageId: string | undefined
+</script>
+
+<style>
+    .spinner {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        height: 100%;
+        width: 100%;
+    }
+</style>
```

Comment:
Why not putting this into the css file?

Reply:
With Svelte we get scoped css out of the box. Meaning that this css will only be applied to the LabelledLoader component. I find that it's easier to reason over css per component instead of everything in one big file (https://svelte.dev/tutorial/styling)

Reply:
I get that. But I prefer to have a consistent way of doing things. So, if we go that way, we should move everything to a component based approach.

Reply:
Agree! Would not be that much work with our current UI. Added https://github.com/KaiOS-Nano-wallet/nano-kaios/issues/50